proprietary/design-tokens/src/component/utrecht/calendar.tokens.json

```diff
@@ -0,0 +1,177 @@
+          "gap": {
+            "value": "5px"
+          }
+        },
+        "days-item-day": {
+          "size": {
+            "value": "44px"
+          },
+          "color": {
+            "value": "{utrecht.color.black}"
+          },
+          "border-width": {
+            "value": ""
+          },
+          "border-color": {
+            "value": "transparent"
+          },
+          "font-family": {
+            "value": ""
+          },
+          "font-size": {
+            "value": "18px"
+          },
+          "font-weight": {
+            "value": ""
+          },
+          "hover": {
+            "background-color": {
+              "value": ""
+            },
+            "color": {
+              "value": "{utrecht.color.blue.40}"
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            }
+          },
+          "out-of-the-month": {
+            "color": {
+              "value": "{utrecht.color.grey.80}"
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "is-today": {
+            "color": {
+              "value": "{utrecht.color.blue.40}"
+            },
+            "font-weight": {
+              "value": "700"
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "emphasis": {
+            "color": {
+              "value": ""
+            },
+            "font-weight": {
+              "value": ""
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "selected": {
+            "color": {
+              "value": ""
+            },
+            "font-weight": {
+              "value": ""
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
+            }
+          },
+          "disabled": {
+            "color": {
+              "value": ""
+            },
+            "font-weight": {
+              "value": ""
+            },
+            "border-width": {
+              "value": ""
+            },
+            "border-color": {
+              "value": "transparent"
+            },
+            "background-color": {
+              "value": ""
```

Comment:
Better to not include `value` at all than to leave it empty, might lead to invalid CSS.